django_ai_assistant/admin.py

```diff
@@ -1,19 +1,70 @@
+from typing import ClassVar
+
 from django.contrib import admin
+from django.contrib.admin.options import InlineModelAdmin
+from django.urls import reverse
+from django.utils.safestring import mark_safe
 
 from django_ai_assistant.models import Message, Thread
 
 
+class MessageProxy(Message):
+    class Meta:
+        proxy = True
+        verbose_name = Message._meta.verbose_name
+        verbose_name_plural = Message._meta.verbose_name_plural
+
+    def __str__(self) -> str:
+        return self.__repr__()
+
+
+class MessageInline(admin.TabularInline):
+    model = MessageProxy
+    extra = 0
+    fields = ("pk", "message_type", "content", "created_at")
+    readonly_fields = fields
+    ordering = ("created_at",)
+    show_change_link = True
+
+    def pk(self, obj):
+        display_text = "<a href={}>{}</a>".format(
+            reverse(
+                f"admin:{Message._meta.app_label}_{Message._meta.model_name}_change", args=(obj.pk,)
+            ),
+            obj.pk,
+        )
+        if display_text:
```

Comment:
Please correct if I'm wrong, but I believe it's not possible for this `if` statement to resolve ever be falsy, I think we can remove it and return directly